packages/amplify-gen2-codegen/src/index.ts

```diff
@@ -142,9 +143,7 @@ export const createGen2Renderer = ({
           new TypescriptNodeArrayRenderer(
             async () => renderFunctions(func),
             (content) => {
-              return fileWriter(content, path.join(dirPath, 'resource.ts'))
-                .then(() => fileWriter('', path.join(dirPath, 'handler.ts')))
-                .catch(console.error);
```

Comment:
This was in the wrong place. It was intended to catch codegen errors but this is while writing code to the files.